src/chef-server-ctl/bin/chef-server-ctl

```diff
@@ -28,13 +29,13 @@ module Omnibus
       d
     end
 
-    CREDENTIAL_ENV = "CHEF_SECRETS_DATA".freeze
+    CREDENTIAL_ENV = "#{ChefUtils::Dist::Infra::SHORT.upcase}_SECRETS_DATA".freeze
```

Comment:
@lbakerchef I did each one of these manually. I used IntelliJ, searched for all instances of `chef` inside strings, and then replaced each one by hand.

This appeared to me to be an environment variable used for configuration, so I assumed (perhaps wrongly) that it should use the product name to be compliant with the trademark policy. If this was a mistake and it's OK for Cinc Server to use `CHEF_SECRETS_DATA` here, then I will remove this change

Reply:
@jgitlin-p21 No worries, whether `sed` (I use it all the time) or IntelliJ or whatever :)  My only point was, this looks like something that will break the code, so I thought it was unintentional.  Someone like @PrajaktaPurohit might know for sure looking at this.  Unless you've already tested this and know it works?

Reply:
It shouldn't break _your_ code because `ChefUtils::Dist::Infra::SHORT.upcase == "CHEF"` so this is not a change.

It might require further changes to _Cinc_ because this will become `"CINC_SECRETS_DATA"`, but in that case I just need to find any other cases where `"CHEF_SECRETS_DATA"` is used and make the same replacement...

Reply:
Thank you @jgitlin-p21! @lbakerchef https://github.com/chef/chef/blob/main/chef-utils/lib/chef-utils/dist.rb#L43 will do the right thing and not make any changes to CREDENTIAL_ENV for Chef builds.